chapters/synchronous.tex

```diff
@@ -1026,7 +1026,7 @@ \subsection{Sub-Clock Inferencing}\label{sub-clock-inferencing}
 64 bit internal representation of numerator and denominator with sign can be used and gives minimum resolution $1.08\times 10^{-19}$ seconds and maximum range $9.22\times 10^{18}$~seconds = $2.92\times 10^{11}$~years.
 \end{nonnormative}
 
-\section{Clocked-Discretized - Continuous-Time Equations in Clocked Partitions}\label{continuous-time-equations-in-clocked-partitions}\label{clocked-discretized-continuous-time-equations-in-clocked-partitions}
+\section{Clocked Partition Equation Solving}\label{continuous-time-equations-in-clocked-partitions}\label{clocked-discretized-continuous-time-equations-in-clocked-partitions}
```

Comment:
I don't think that clearly represents the contents - the section is focused on using solverMethod to discretize equations.
Originally it was "Continuous-Time Equations in Clocked Partitions", and then changed to add "Clocked-Disretized - ", since "continuous-time" wasn't fully correct.
I would thus prefer "Clocked-Discretized" or "Clocked-Discretized Partition".
```suggestion
\section{Clocked-Discretized Partition}\label{continuous-time-equations-in-clocked-partitions}\label{clocked-discretized-partition}
```

Reply:
Before committing the suggestion, I'd like to get confirmation that it really is a better name considering that it is also under this heading that _clocked discrete-time partition_ is defined.

(And I guess there shouldn't be any hyphen in _clocked discretized partition_, right?)

Reply:
There's another discussion of having better names for this #3091

Reply:
Even if clocked discrete-time partition is defined here, it is just as the complement to clocked discretized; which is the main focus of the section - but as noted in #3091 it is not defined in a good way.